README.md

```diff
@@ -168,6 +169,92 @@ export PATH=$(HOME)/local/bin:$PATH
 
     $ sbt pack-archive
 
+### Find duplicated classes in dependencies
```

Comment:
I feel this explanation is a little bit long for README.md. Could you create a new file (e.g., USAGE.md), then move this section to this file?)